products/compute/api.yaml

```diff
@@ -5841,6 +5841,170 @@ objects:
+        name: 'name'
+        required: true
+        description: |
+          The name of the resource, provided by the client when initially creating
+          the resource. The resource name must be 1-63 characters long, and comply
+          with RFC1035. Specifically, the name must be 1-63 characters long and
+          match the regular expression [a-z]([-a-z0-9]*[a-z0-9])? which means the
+          first character must be a lowercase letter, and all following characters
+          must be a dash, lowercase letter, or digit, except the last character,
+          which cannot be a dash.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'snapshotSchedulePolicy'
+        description: |
+          Policy for creating snapshots of persistent disks.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'schedule'
+            description: |
+              Contains one of an `hourlySchedule`, `dailySchedule`, or `weeklySchedule`.
+            required: true
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'hourlySchedule'
+                description: |
+                  The policy will execute every nth hour starting at the specified time.
+                properties:
+                  - !ruby/object:Api::Type::Integer
+                    name: 'hoursInCycle'
+                    description: |
+                      The number of hours between snapshots.
+                    required: true
+                  - !ruby/object:Api::Type::String
+                    name: 'startTime'
+                    description: |
+                      Time within the window to start the operations.
+                      It must be in format "HH:MM",
+                      where HH : [00-23] and MM : [00-00] GMT.
+                    required: true
+              - !ruby/object:Api::Type::NestedObject
+                name: 'dailySchedule'
+                description: |
+                  The policy will execute every nth day at the specified time.
+                properties:
+                  - !ruby/object:Api::Type::Integer
+                    name: 'daysInCycle'
+                    description: |
+                      The number of days between snapshots.
+                    required: true
+                  - !ruby/object:Api::Type::String
+                    name: 'startTime'
+                    description: |
+                      This must be in UTC format that resolves to one of
+                      00:00, 04:00, 08:00, 12:00, 16:00, or 20:00. For example,
+                      both 13:00-5 and 08:00 are valid.
+                    required: true
+              - !ruby/object:Api::Type::NestedObject
+                name: 'weeklySchedule'
+                description: |
+                  Allows specifying a snapshot time for each day of the week.
+                properties:
+                  - !ruby/object:Api::Type::Array
```

Comment:
Does this need to be a set? i.e. if I specified a list of 5 days, does GCP always return it in the same order?

Reply:
Good question - I just checked and GCP returns it in the same order.

Reply:
Do we want to treat it as a set anyways though? I'd lean towards making this a set even if GCP returns the same order, since I'm not sure if users who change the order of them won't want to recreate the resource.

(requires `is_set: true`, the default hash function should just be fine since they are both strings returned in same format from API as given)

Reply:
Done.